release/release_tests.yaml

```diff
@@ -847,10 +847,7 @@
   cluster:
     byod:
       type: gpu
-      pip:
-        - myst-parser==1.0.0
-        - myst-nb==1.0.0
-        - jupytext==1.13.6
```

Comment:
Do we not need `jupytext` anymore? It's not in `byod_gptj_test.sh` either

Reply:
its in the base.